src/app/common/psbt/use-psbt-request-params.ts

```diff
@@ -0,0 +1,48 @@
+import { useMemo } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+import * as btc from '@scure/btc-signer';
+
+import { undefinedIfLengthZero } from '@shared/utils';
+
+import { useRejectIfLedgerWallet } from '@app/common/rpc-helpers';
+
+import { useDefaultRequestParams } from '../hooks/use-default-request-search-params';
+
+export function usePsbtRequestSearchParams() {
+  const [searchParams] = useSearchParams();
+  const { origin, tabId } = useDefaultRequestParams();
+  const requestToken = searchParams.get('request');
+  return useMemo(
+    () => ({
+      origin,
```

Comment:
Does the tabId of `1` or `0` mean something? Maybe we could put them in a well named `enum`? 

```
enum PSBT_TABS: { 
 I_AM: 0,
A_TAB: 1
}
```

Reply:
@pete-watters tabId is a auto-generated value from the browser we don't know upfront

Reply:
OK, so it's browser tab, thanks!